modules/migration-understanding-resources-mtc-migrates.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * migration/migrating_3_4/migrating-application-workloads-3-4.adoc
+// * migration/migrating_4_1_4/migrating-application-workloads-4-1-4.adoc
+// * migration/migrating_4_2_4/migrating-application-workloads-4-2-4.adoc
+
+[id='about-resources-mtc-migrates_{context}']
+= About the resources migrated
+[role="_abstract"]
+
+You can use the {mtc-full} ({mtc-short}) to migrate the following types of resources:
+
+* Namespaces specified in migration plans. Migration plans have the granularity of a namespece.
+
+* Namespace-scoped resources. When a namespace is selected for migration, then all the objects and resources (services, pods, etc.) pertaining to that namespace are also selected for migration. Additionally, if a resource scoped by a namespace depends on a resource that is scoped by a cluster, then the resource scoped by the cluster is also migrated by the  {mtc-full} Operator.
```

Comment:
Rewrite to remove fluff. Avoid using parentheses. They can be problematic for localization.

Remove "then". It does not add anything to the meaning, AFAIK.

The resources are not migrated by the operator. The SME is using 'operator' to refer to the VMIO component. It's just MTC to the user.